clang-tools-extra/clang-tidy/cppcoreguidelines/MissingStdForwardCheck.cpp

```diff
@@ -53,18 +53,76 @@ AST_MATCHER(ParmVarDecl, isTemplateTypeParameter) {
          FuncTemplate->getTemplateParameters()->getDepth();
 }
 
+AST_MATCHER_P(NamedDecl, hasSameNameAsBoundNode, std::string, BindingID) {
+  const auto &Name = Node.getNameAsString();
```

Comment:
getNameAsString looks to be deprecated, use getName or even better getIdentifier()->getName() and check that getIdentifier is not null

Reply:
Add release notes and fix code as your suggestion.